src/components/TodoItem.tsx

```diff
@@ -0,0 +1,66 @@
+import cn from 'classnames';
+import { useState } from 'react';
+import { useTodoContext } from '../context/TodoContext';
+import { deleteTodo } from '../api/todos';
+import { Todo } from '../types/Todo';
+import { Error } from '../types/Error';
+
+interface Props {
+  key: number;
```

Comment:
The 'key' prop should not be used in the Props interface since it is a reserved property in React. Please remove 'key' from Props.